### PR TITLE
Add music memories to map

### PR DESCRIPTION
**Pull request by TPGamesNL**
Adds music memories to the map. Can easily be modified to include e.g. concert memories (extend PostOverlay).

Example:
image

Currently displays the photo of the post itself, will likely be changed later to the photo of the song (e.g. album cover). The delayed image getting is also taken into account and will require little modifications (`PostOverlay#setImage`).

Since we don't have a screen that displays a post (such as music memory) yet, the click handler is not implemented yet. Implementing this should also require few changes.

### CODE REVIEW

File: app/src/main/java/com/example/musicmap/screens/map/MapFragment.java [hunk 1]
```diff
@@ -96,11 +105,22 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
...
+                    sharedPreferences.getLong(SHARED_PREFERENCE_ZOOM, 0)));
+            // SharedPreferences cannot store double directly, use long instead
+
+            IGeoPoint center = new GeoPoint(
+                    Double.longBitsToDouble(sharedPreferences.getLong(SHARED_PREFERENCE_CENTER_LATITUDE, 0)),
+                    Double.longBitsToDouble(sharedPreferences.getLong(SHARED_PREFERENCE_CENTER_LONGITUDE, 0))
+            );
+            mapView.getController().setCenter(center);
+        } else {
+            // Set initial view to map of Netherlands
```
Comment (Pradyuman7) [inline app/src/main/java/com/example/musicmap/screens/map/MapFragment.java:120]: Should we keep the initial zoom to The Netherlands? We could make it the location of user.

Comment (TPGamesNL) [reply app/src/main/java/com/example/musicmap/screens/map/MapFragment.java:120]: The problem with having it be the location of the user is that getting this location takes time; it's not already there when we load the map. So, we'd either have to delay loading the map or we'd have to move the map when we actually get the GPS location, which is after the map has been loaded. Both aren't ideal, which is why I chose to have a standard initial setting instead

Comment (Pradyuman7) [reply app/src/main/java/com/example/musicmap/screens/map/MapFragment.java:120]: Hm, if getting this location takes time, can we maybe put the location fetching to the HomeActivity which loads as soon as user is logged in. This way we can probably load the location before user opens the map view.

But, anyway this is very low priority stuff imo and is fine as we have now.

**Review by Pradyuman7 — APPROVED** [review]
LGTM

**Pull request merged**